src/composer.json

```diff
@@ -58,10 +59,11 @@
     "config": {
         "allow-plugins": {
             "dealerdirect/phpcodesniffer-composer-installer": true,
-            "liborm85/composer-vendor-cleaner": true
+            "liborm85/composer-vendor-cleaner": true,
+            "php-http/discovery": true
         },
         "platform": {
-            "php": "7.2"
+            "php": "7.3"
```

Comment:
Fix this, and figure out why composer causes such headaches.

Reply:
@sergunich Do we need this to be 7.2 instead of 7.3? I'm getting composer conflicts unrelated to any requirements I've added whenever I do a composer update. (The conflicts have to do with symfony and the php version.)